CMakeLists.txt

```diff
@@ -411,10 +419,12 @@ foreach(target devilution ${BIN_TARGET})
   endif()
 
   if(3DS)
-    target_compile_definitions(${target} PRIVATE 3DS)
-    target_link_libraries(${target} PRIVATE 3ds::ctrulib
-      # Most of these shouldn't be necessary.	
-      -lSDL_ttf -lfreetype -lSDL_mixer -lvorbisfile -lvorbis -logg -lmikmod -lmad -lSDL -lpng -lbz2 -lz -lcitro3d -lctru -lm)
+    target_link_libraries(${target} PRIVATE ${BZIP2_LIBRARIES})
+    target_include_directories(${target} PRIVATE ${BZIP2_INCLUDE_DIR})
+    target_link_libraries(${target} PRIVATE ${VORBIS_LIBRARIES})
+    target_include_directories(${target} PRIVATE ${VORBIS_INCLUDE_DIRS})
+    target_link_libraries(${target} PRIVATE 3ds::ogg 3ds::mikmod 3ds::mad 3ds::freetype 3ds::png)

```

Comment:
In case it helps only `freetype` is required, the rest are unused and should be optional for SDL.